app/js/request.js

```diff
@@ -26,8 +26,37 @@ var requestModule = (function(){
     });
   };
 
+  var openWebsocket = function(url, keepAliveInterval, callbacks){
+    socket = new WebSocket(url);
+    socket.onopen = function(e){
+      verifyClient(socket);
+      callbacks.open(e);
+    };
+    socket.onclose = callbacks.close;
+    socket.onmessage = callbacks.message;
+
+    setInterval(function(){
+      keepAlive(socket, keepAliveInterval);
+    }, keepAliveInterval);
+  };
+
+  var keepAlive = function(socket, interval){
+    sendSocketMessage(socket);
+  };
+
+  var sendSocketMessage = function(socket, message){
+    var parcel = {
+      csrf: getCsrfToken(),
```

Comment:
This should ultimately be unnecessary when the websocket server is fully configured.